FirebaseAuth/Tests/SampleSwift/SwiftApiTests/PhoneAuthTest.swift

```diff
@@ -0,0 +1,56 @@
+//
```

Comment:
Please add Google copyrights
You can refer to https://github.com/firebase/firebase-ios-sdk/blob/main/FirebaseAuth/Tests/SampleSwift/SwiftApiTests/PhoneMultiFactorTests.swift or other files in the repo